OtripleS.Web.Api/Services/TeacherContacts/TeacherContactService.cs

```diff
@@ -44,6 +44,11 @@ public partial class TeacherContactService : ITeacherContactService
 			return storageTeacherContacts;
 		});
 
+		public ValueTask<TeacherContact> RetrieveTeacherContactByIdAsync(Guid teacherId, Guid contactId)
+		{
+			throw new NotImplementedException();
```

Comment:
If you have the capacity please implement this.
Otherwise @viralpandya  if you can please take this task.

Reply:
I will take this one.

Reply:
ok